app/webpack/js/json_api.js

```diff
@@ -0,0 +1,42 @@
+import { get as ajaxGet } from "@js/mau_ajax";
+import { ArtPiece } from "@models/art_piece.model";
+import { Artist } from "@models/artist.model";
+import { Studio } from "@models/studio.model";
+import { camelizeKeys } from "humps";
+
+const get = (url) => ajaxGet(url).then((data) => camelizeKeys(data));
+
+export const jsonApi = {
```

Comment:
This is the beginning of an api connection that does not care about `react` or `angular`.  A bit sadly, it's calling `mau_ajax` which uses `jquery.ajax` but we can move to `fetch` or something else later.